docs/examples/201/policy-with-initiative-definition-and-assignment/main.bicep

```diff
@@ -1,3 +1,5 @@
+targetScope = 'subscription'
```

Comment:
isn't policy def and assignment valid at any scope?

Reply:
Our type definitions have "policySetDefinitions" as MG & sub only: https://github.com/Azure/bicep-types-az/blob/main/generated/resources/microsoft.authorization/2018-03-01/types.md#resource-microsoftauthorizationpolicysetdefinitions2018-03-01

The API docs seem to support this:
https://docs.microsoft.com/en-us/rest/api/resources/policysetdefinitions/createorupdate
https://docs.microsoft.com/en-us/rest/api/resources/policysetdefinitions/createorupdateatmanagementgroup

Reply:
Ah, maybe it's only assignments that are valid at RG. LGTM.